packages/suite/package.json

```diff
@@ -39,7 +39,6 @@
         "@suite-common/wallet-config": "workspace:*",
         "@suite-common/wallet-constants": "workspace:*",
         "@suite-common/wallet-core": "workspace:*",
-        "@suite-common/wallet-graph": "workspace:*",
```

Comment:
I am curious, what does this mean exactly? Is it somehow connected with NX?

Reply:
What do you mean? That `workspace:*`?

Reply:
> What do you mean? That `workspace:*`?

yes

Reply:
It related to yarn workspaces. It's layer of defence again supply chain attack typo squating. Someone might try to publish package to npm with name for example `@suite-common/grahp` and wait if someone will do typo mistake and use it. If you are using this `workspace:*` local package resolution is enforced and if not available local, it will fail.

https://yarnpkg.com/features/workspaces#workspace-ranges-workspace